peoplepage.html

```diff
@@ -30,9 +46,19 @@
         .searchbox {
             margin-left: 750px;
```

Comment:
same here change to margin-left: auto to prevent clipping

Reply:
done